src/hub/action_form.ts

```diff
@@ -14,7 +20,9 @@ export interface ActionFormField {
   label?: string
   description?: string
   default?: string
-  type?: "string" | "textarea" | "select"
+  type?: "string" | "textarea" | "select" | "set_params_link" | "oauth_link"
```

Comment:
Are there situations where we return a form that has fields AND a `"set_params_link"` or  `"oauth_link"`?

If there's no use case for that, it would be a cleaner API to not put these concepts into the `fields` section of the form, but maybe into a new form key that is mutually exclusive with `fields`.

Reply:
oh good catch, "set_params_link" is cruft that needs to be removed